src/main.css

```diff
@@ -37,3 +37,12 @@
     display: none;
   }
 }
+
+/** Components */
+
+:is(.dark [role="alert"]),
+[role="alert"] {
+  a {
+    color: inherit;
```

Comment:
I thought flowbite took care of link colors in alerts?

Reply:
Well, it didn't. Maybe it's specific to the fact I'm using `<Link />` here instead of just regular <a /> that might come from a markdown transformation.